src/main/kotlin/mate/academy/Main.kt

```diff
@@ -0,0 +1,10 @@
+package mate.academy
+
+fun main() {
+    val ball1 = Lottery()
+    println(ball1.getRandomBall().toString())
+    val ball2 = Lottery()
+    println(ball2.getRandomBall().toString())
+    val ball3 = Lottery()
+    println(ball3.getRandomBall().toString())
+}
```

Comment:
Class "Lottery" is just a fabric, which will produce material
We don't need three Lottery to get three balls
It is the same, as we would have 3 coffee makers at home to make 3 coffee, who need 3 coffee makers?
In code we are just losing poductivity and memory
It should be like that:

 val lottery = Lottery()
 println(lottery.getRandomBall().toString())
 println(lottery.getRandomBall().toString())
 println(lottery.getRandomBall().toString())
 
 But code is repeated, so better to use for cycle to make it more beautiful.